relay/config/config.go

```diff
@@ -73,6 +76,12 @@ func (c *Config) Verify() error {
 	if c.DockerEnabled() == false && c.NativeEnabled() == false {
 		return errorNoExecutionEngines
 	}
+	if c.ManagedDynamicConfig == true && c.DynamicConfigRoot == "" {
+		return errorMissingDynamicConfigRoot
+	}
+	if c.ManagedDynamicConfig == true {
+		c.DynamicConfigRoot = path.Join(c.DynamicConfigRoot, "managed")
```

Comment:
This means we can't ever have a bundle named "managed", correct? We should guard against that in Cog, then (not a big deal; we already have a few other protected bundle names, and the rest of the app deals with them fine; we just need to add a new string for it to guard against)